features/levering-koopsom.feature

```diff
@@ -4,18 +4,18 @@ Functionaliteit: Koopsommen worden alleen geleverd in combinatie met koopjaar al
                  Het element indicatieMetMeerObjectenVerkregen (van koopsom) maakt geen deel uit van deze combinatie.
 
     Achtergrond:
-        Gegeven de registratie BRK kent de onroerendezaak met kadastraalonroerendezaakidentificatie: NL.IMKAD.KadastraalObject.76870487970000,
+        Gegeven de registratie BRK kent de onroerendezaak met kadastraalonroerendezaakidentificatie: 76870487970000,
         En waarvan de volgende velden gevuld zijn: identificatie, type, kadastraleaanduiding, koopsom (van koopsom), koopjaar (van koopsom), aardCultuurbebouwd
-        En de onroerendezaak met kadastraalonroerendezaakidentificatie: NL.IMKAD.KadastraalObject.76870488070000,
+        En de onroerendezaak met kadastraalonroerendezaakidentificatie: 76870488070000,
         En waarvan de volgende velden gevuld zijn: identificatie, type, kadastraleaanduiding, koopsom (van koopsom), aardCultuurbebouwd
 
     Scenario: Onroerendezaak (met aardCultuurBebouwd & koopsom, welke koopsom en koopjaar bevat) wordt opgevraagd zonder fields parameter
-        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/76870487970000
         Dan worden alle velden van koopsom & aardCultuurBebouwd teruggegeven
         En wordt de volgende json response teruggegeven:
         """
         {
-            "identificatie": "NL.IMKAD.KadastraalObject.76870487970000",
+            "identificatie": "76870487970000",
```

Comment:
Dient sinds #510 ook "domein" in dit response voorbeeld terug te komen?
```json
"domein": "NL.IMKAD.KadastraalObject"
```

Reply:
toegevoegd